nix/overlay.nix

```diff
@@ -9,4 +9,5 @@ self: super: {
   control-plane = super.callPackage ./pkgs/control-plane { inherit allInOne incremental tag; };
   openapi-generator = super.callPackage ./pkgs/openapi-generator { };
   xfsprogs_5_16 = (import (super.sources).nixpkgs-22_05 { }).xfsprogs;
+  rdmacore = (import (super.sources).nixpkgs-22_05 { }).rdma-core;
```

Comment:
why do we need to pull a special rdma-core?

Reply:
I mean rather than just taking the default one from nixpkgs

Reply:
There is no special requirement. I think I just followed example from `xfsprogs` and added here. Are you suggesting adding package name to shell.nix and cargo-package.nix, and not pulling from a specific nix channel? If that also works, should be ok.

Reply:
Yeah just remove it from here and use it as any other package

Reply:
modified nix scripts.